client/src/views/Login/Login.js

```diff
@@ -0,0 +1,253 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+import {
+  Typography,
+  TextField,
+  Link as HyperLink,
+  InputAdornment,
+  IconButton,
+} from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { Button } from '../../components/Generic';
+import { Visibility, VisibilityOff } from '../../components/icons';
+import style from './Login.module.css';
+
+const EmailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
```

Comment:
tests for this regex?

Reply:
Tests have been added in the Api integration PR.